learn/react-native/disable-device-screencapture.md

```diff
@@ -0,0 +1,98 @@
+apps. By enabling or disabling screenshots programmatically, developers can enhance security and privacy. 
+Preventing users from capturing sensitive content, such as confidential data or premium resources, safeguards user 
+information and protects intellectual property. With a seamless integration within the Expo ecosystem, the ScreenCapture
+plugin ensures a smooth implementation process. Developers can easily enforce screenshot restrictions, giving them greater 
+control over app security and privacy. By leveraging Expo's ScreenCapture plugin, developers can create more secure and 
+privacy-conscious apps, delivering a safer and more confidential user experience.
+
+## Adding ScreenCapture to your WaveMaker React Native App
+
+ScreenCapture plugin can be installed in few steps in WaveMaker. Please refer to this [page](https://docs.wavemaker.com/learn/react-native/third-party-expo-plugins#expo)
+on how to install a plugin.
+
+## Using ScreenCapture in your WaveMaker App
+
+ScreenCapture plugin can be leveraged in your WaveMaker app through the following ways:
+
+- Enabling and Disabling ScreenCapture on Demand
+- Disabling ScreenCapture on App Launch
+
+### Enabling and Disabling ScreenCapture on Demand
+
+Some apps may require the ability to enable or disable screen capture only in certain situations i.e. on demand.
+For example, a video streaming app may want to allow screenshots when the user is watching a free video, but disable for
+content that requires a paid subscription.
+
+For our example, we will create a simple app that allows users to enable or disable screenshots on click of a button. 
+The following is the mockup of the app:
+
+![ScreenCaptureOnDemand](/learn/assets/ScreeCaptureOnDemand.png)
+
+The following is the code snippet for the above mockup:
+
+```javascript
+const ScreenCapture = require('expo-screen-capture');
+/* perform any action on widgets/variables within this block */
+Page.onReady = function() {
+};
+
+Page.enableCapture = async function($event, widget) {
+    await ScreenCapture.allowScreenCaptureAsync();
+    Page.Widgets.enable.disabled = true;
+    Page.Widgets.disable.disabled = false;
+    alert("Screenshot option is enabled");
+
+};
+Page.disableCapture = async function($event, widget) {
+    await ScreenCapture.preventScreenCaptureAsync();
+    Page.Widgets.enable.disabled = false;
+    Page.Widgets.disable.disabled = true;
+    alert("Screenshot option is disabled");
+};
+```
+
+### Disabling Screenshots on App Launch
+
+Some apps may require the ability to disable screen capture on app launch. For example, a banking app may want to disable
+for all screens to prevent sensitive information from being captured. In such cases, the ScreenCapture plugin can be leveraged
+as shown below:
+
+In your WaveMaker project navigate to `src/main/webapp/app.js` file and add the following code snippet:
```

Comment:
In your WaveMaker project navigate to the src/main/webapp/app.js file and add the following code snippet, which imports the plugin into the app.

Reply:
Updated